config/locales/en.yml

```diff
@@ -462,6 +462,9 @@ en:
       how_can_i_submit_my_event_answer: "<p>To submit your event, click on \"Submit Event\" and fill out all the details. Your personal information (name and email) will not be public and is only asked so that we can contact you.</p>"
       how_can_i_add_tags: "How can I add a new tag to my event?"
       how_can_i_add_tags_answer: "<p>We currently only allow site administrators to add new tags; if you'd like to tag your event with a tag that is currently not available from the list, please drop us a line at %{travis_email}. We will try to process your request as quickly as possible and add your chosen tag to your event submission.</p>"
+      why_provide_a_link: "Why do I have to provide a link when I manage application and selection myself?"
```

Comment:
Would add a “the” here, meaning: “Why do I have to provide a link when I manage the application and selection myself?”